### PR TITLE
Issue#14637: send empty string to device instead of null when custome…

### PR DESCRIPTION
**Pull request by JesseCodeBones**
…r replace the content with empty value

### CODE REVIEW

File: lib/protocol/routes.js [hunk 1]
```diff
@@ -581,7 +581,7 @@ const METHOD_MAP = {
       // We want to either a value (old JSONWP) or a text (new W3C) parameter,
       // but only send one of them to the command (not both).
       // Prefer 'value' since it's more backward-compatible.
-      makeArgs: (jsonObj) => [jsonObj.value || jsonObj.text],
+      makeArgs: (jsonObj) => (jsonObj.value || jsonObj.text) == null ? [''] : [jsonObj.value || jsonObj.text],
```
Comment (mykola-mokhnach) [inline lib/protocol/routes.js:584]: `[jsonObj.value || jsonObj.text || '']` should work

**Comment by mykola-mokhnach** [comment]
@JesseCodeBones  Please sign the CLA

File: lib/protocol/routes.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/protocol/routes.js:584]: or even better `[jsonObj.value ?? jsonObj.text]`

Comment (JesseCodeBones) [reply lib/protocol/routes.js]: Hi Mykola, 
I chosed the first change, if the `value` and `text` are both null value, the null value could also send to device.
so it seems first chose is better.

Regards

Comment (mykola-mokhnach) [reply lib/protocol/routes.js]: then it would make sense to still replace `||` with `??`

**Comment by KazuCocoa** [comment]
The CLA must be the same as your commit's email address

**Comment by JesseCodeBones** [comment]
rebuildpr

**Comment by JesseCodeBones** [comment]
@KazuCocoa  @mykola-mokhnach  
I resigned the CLA, but the check is not pass, how to retrigger the checks for Pull Request? Thank you.

File: lib/protocol/routes.js (hunk 1 above)
Comment (jessechen0319) [reply lib/protocol/routes.js]: @mykola-mokhnach finished the change, please kindly review, Thank you.

**Pull request merged**